hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmKeyInfo.java

```diff
@@ -772,6 +793,7 @@ public boolean isKeyInfoSame(OmKeyInfo omKeyInfo, boolean checkPath,
         replicationConfig.equals(omKeyInfo.replicationConfig) &&
         Objects.equals(metadata, omKeyInfo.metadata) &&
         Objects.equals(acls, omKeyInfo.acls) &&
+        Objects.equals(ownerName, omKeyInfo.ownerName) &&
```

Comment:
ownerName comparision can be based on flag, as this is used for snapdiff for snapshot, where ownername can be different but key will be same

Reply:
Done.